src/defold/Sound.hx

```diff
@@ -100,7 +100,7 @@ extern class Sound {
         @param url the sound that should play
         @param play_properties optional table with properties
     **/
-    static function play<T>(url:HashOrStringOrUrl, ?play_properties:SoundMessagePlaySound, ?complete_function:(T,Hash,SoundMessageSoundDone,Url)->Void):Void;
+    static function play<T>(url:HashOrStringOrUrl, ?play_properties:SoundMessagePlaySound, ?complete_function:T->Hash->SoundMessageSoundDone->Url->Void):Void;
```

Comment:
Maybe we should remove support for Haxe 3, as Haxe 4 is now properly released and is better in every way especially with regard to Lua support...